wrappers/services.go

```diff
@@ -876,47 +911,21 @@ func EnsureSnapServices(snaps map[*snap.Info]*SnapServiceOptions, opts *EnsureSn
 	return context.reloadModified()
 }
 
-// AddSnapServicesOptions is a struct for controlling the generated service
+// generateSnapServicesOptions is a struct for controlling the generated service
 // definition for a snap service.
-type AddSnapServicesOptions struct {
+type generateSnapServicesOptions struct {
 	// VitalityRank is the rank of all services in the specified snap used by
 	// the OOM killer when OOM conditions are reached.
 	VitalityRank int
 
-	// QuotaGroup is the quota group for all services in the specified snap.
+	// QuotaGroup is the quota group of the snap, of which services we are
+	// generating
```

Comment:
this is not always true as this also gets set to the group for a specific service and not for a full snap

Reply:
Yes good point, I mixed it up with the other Quota Group member in our EnsureSnapServiceOptions. Thanks for catching this